### DISCUSSION TITLE
Is an opt-in option for flashing content sufficient to meet SC 2.3.1?

### DISCUSSION THREAD

**Discussion by mraccess77** [w3c/wcag#4046] [Q&A] [answered]
SC 2.3.1 indicates that opt out mechanisms are not allowed - this could imply that opt-in mechanisms are acceptable.  Is this sufficient?

**Comment by JAWS-test** [comment, marked as answer]
I would assume that an opt-in mechanisms is allowed for every SC, as long as the label of the element for activation is correct, i.e. it is clear what is happening (for 2.3.1 as a warning)
